scripts/dockutil

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/python
```

Comment:
What should shebang be?  #!/usr/bin/env python (if it also works in python2) or #!/usr/bin/env python3 ? or something else?

Reply:
Great question. I didn't have an answer to that question so I axed it from the PR.

Ideally it would be interesting to test this with official python3 package and then use #!/usr/bin/env python3 but you could run into an issue where someone has a different python installed in their user path. dockutil isn't typically ran in the user space so that shebang could be a bad assumption. 

My gut feeling is people triggering dockutil should use it without a shebang and then invoke it against the python3 of their choice. So for myself I do /Library/InstallApplications/Python.framework/blahblahblah/bin/python3 /path/to/shebangless/dockutil 

Everyone writing python3 is going to run into this. If you want to be authoritative, it's highly likely you will have to include your own python framework for others to use and it will increase your support burden. 

Wish I had a better answer.